kernel/aster-nix/src/fs/utils/page_cache.rs

```diff
@@ -98,18 +99,30 @@ impl PageCacheManager {
     pub fn evict_range(&self, range: Range<usize>) -> Result<()> {
         let page_idx_range = get_page_idx_range(&range);
         let mut pages = self.pages.lock();
+
+        //TODO: When there are many pages, we should submit them in batches of folios rather than all at once.
+        let mut waiters = vec![];
```

Comment:
Specify the type here makes the code more readable.
```
let mut ides_and_waiters: Vec<(usize, BioWaiter)> = vec![];
```

Reply:
FIXED